lib/web/servers.go

```diff
@@ -90,10 +90,14 @@ func (h *Handler) getDesktopHandle(w http.ResponseWriter, r *http.Request, p htt
 
 	desktopName := p.ByName("desktopName")
 
-	windowsDesktop, err := clt.GetWindowsDesktop(r.Context(), desktopName)
+	windowsDesktops, err := clt.GetWindowsDesktopsByName(r.Context(), desktopName)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	windowsDesktops = types.DeduplicateDesktops(windowsDesktops)
```

Comment:
In this case, I don't think you need to deduplicate since they're all the same from the UI's perspective.

As long as you got at least 1 desktop, you should be able to just

```
return ui.MakeDesktop(windowsDesktops[0]), nil
```